modules/con-vuln-sources.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+// * cloud_service/acscs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="con-vuln-sources_{context}"]
+= Vulnerability data sources
+
+Sources for vulnerabilities depend on the scanner that is used in your system. {product-title-short} contains two scanners: the StackRox Scanner and Scanner V4. The StackRox Scanner is the default scanner and is deprecated beginning with release 4.6. Scanner V4 was introduced in release 4.4 and is the recommended scanner.
```

Comment:
nit: should we say "recommended image scanner" since it doesn't handle anything else yet (other than RHCOS nodes, but that's in tech preview)?